ui/src/features/Canvas/index.tsx

```diff
@@ -32,6 +32,7 @@ const snapGrid: SnapGrid = [gridSize, gridSize];
 const proOptions: ProOptions = { hideAttribution: true };
 
 type Props = {
+  isSharedCanvas: boolean;
```

Comment:
```suggestion
  isSharedCanvas?: boolean;
```
If you make this optional, you don't need to pass anything when it is not a shared canvas, and then you can reduce `isSharedCanvas={true}` to just `isSharedCanvas`. Keeps things cleaner when using `Canvas`